functions/Test-DbaSqlLoginWeakPassword.ps1

```diff
@@ -0,0 +1,131 @@
+            By default it will test for empty password and the same password as username.
+
+        .PARAMETER SqlInstance
+            The SQL Server instance you're checking logins on. You must have sysadmin access and server version must be SQL Server version 2008 or higher.
+
+        .PARAMETER SqlCredential
+            Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:
+
+            $scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.
+
+            Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+
+            To connect as a different Windows user, run PowerShell as that user.
+
+        .PARAMETER SqlPass
+            Specifies a list of passwords to include in the test for weak passwords.
+
+        .PARAMETER Path
+            Specifies a csv file path of passwords to include in the test for weak passwords.
+            The CSV file has no header, it should just have the passwords you want to check.
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Author: Peter Samuelsson
+
+            dWebsite: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: MIT https://opensource.org/licenses/MIT
+
+        .LINK
+            https://dbatools.io/Test-DbaSqlLoginWeakPassword
+
+        .EXAMPLE
+            Test-DbaSqlLoginWeakPassword -SqlInstance Dev01
+
+            Test all SQL logins that the password is null or same as username on SQL server instance Dev01
+
+        .EXAMPLE
+            Test-DbaSqlLoginWeakPassword -SqlInstance Dev01 -SQLPass Test1,test2
+
+            Test all SQL logins that the password is null, same as username or Test1,Test2 on SQL server instance Dev01
+
+        .EXAMPLE
+            Test-DbaSqlLoginWeakPassword -SqlInstance Dev01 -Path c:\dbatools\weakpassword.cvs
+
+            Test all SQL logins that the password is null, same as username and defined in the csv file on SQL server instance Dev01
+
+    #>
+    [CmdletBinding()]
+    Param (
+        [parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer", "SqlServers")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [String[]]$SQLPass,
+        [String]$Path,
+        [Alias('Silent')]
```

Comment:
You can remove the alias, this was only needed for old command...new ones do not require it.